File: JSBASIC/183_sum_negative.js
// 183	183_sum_negative.js	Calculate the sum of only negative numbers in an array.

let arr = [5, -3, 8, -1, 12, -7, 4];
let sum = 0;
for (let i = 0; i < arr.length; i++) {
    if (arr[i] < 0) sum += arr[i];
}
console.log("Array:", arr);
console.log("Sum of negative numbers:", sum);

File: JavaBasic/183_sum_negative.java
// 183	183_sum_negative.java	Calculate the sum of only negative numbers in an array.
public class _183_sum_negative {
    public static void main(String[] args) {
        int[] arr = {5,-3,8,-1,12,-7,4};
        int sum = 0;
        for (int x:arr) if(x<0) sum+=x;
        System.out.println("Sum of negatives: " + sum);
    }
}
